.github/workflows/ros_build.yaml

```diff
@@ -10,38 +10,7 @@ concurrency:
   group: "${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}"
   cancel-in-progress: true
 jobs:
-  clang-format-lint:
```

Comment:
Should we add inside the repo our clang -format file?

Reply:
It should be there already! The `.clang-format` and the `CPPLINT.cfg`